### PR TITLE
Image rename

### PR DESCRIPTION
**Pull request by jswank**
This PR renames the docker images to ghcr.io/infinite-industries/api-server and ghcr.io/infinite-industries/web-portal.

### CODE REVIEW

File: .github/workflows/publish-container-images.yml [hunk 1]
```diff
@@ -13,18 +13,63 @@ on:
...
+
 jobs:
   publish-ops-server:
     runs-on: ubuntu-latest
     defaults:
       run:
         shell: bash
         working-directory: ops-server
+    env:
+      IMAGE_NAME: infinite-industries/ops-server
```
Comment (chriswininger) [inline .github/workflows/publish-container-images.yml]: I don't quite understand what ops-server is?

File: DEPLOY.MD [hunk 2]
```diff
@@ -16,10 +16,10 @@ Typically, for a staging release you will use the `development` branch. For prod
 *Usually the "Use workflow from" will be the same as the "commit or tag to use" field. The only reason this might very is
 if you are testing changes to the github actions its self.*
 
-Run the action. When complete a new versions of the images tagged with the hash/git tag should be available on docker hub:
+Run the action.  When complete a new versions of the images tagged with the hash/git tag should be available on ghcr.io
 
-* [front end image](https://hub.docker.com/repository/docker/chriswininger/infinite-web-portal/general)
-* [backend image](https://hub.docker.com/repository/docker/chriswininger/infinite-api-server/general)
+* [infinite-industries/web-portal](https://ghcr.io/infinite-industries/web-portal) - the frontend image
```
Comment (chriswininger) [inline DEPLOY.MD:21]: I like that this gets us off of my personal dockerhub, thx for working through this :-)

File: ansible/db-backup/tasks/main.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@
     path: ~/backups
     state: directory
 - name: fetch (or update) the container
-  command: docker pull ghcr.io/infinite-industries/ops:latest
+  command: docker pull ghcr.io/infinite-industries/ops-server:{{ image_version }}
```
Comment (chriswininger) [inline ansible/db-backup/tasks/main.yml:11]: ok now i see that ops-server was likely just a rename from the image formerly called ops, but what was ops where where/how is it built and published?

File: ansible/docker-files/keys/staging-1nfinite.pem
Comment (chriswininger) [inline ansible/docker-files/keys/staging-1nfinite.pem:1]: What exactly changed with the staging-pem?

File: ansible/group_vars/prod/secrets
Comment (chriswininger) [inline ansible/group_vars/prod/secrets:1]: guessing this is where you added the map box secret? This change probably goes away when you merge development back into the branch?

**Review by chriswininger — COMMENTED** [review]
left a couple of questions, looks like some good stuff. I know this is still marked draft just figured I'd have a look

File: .github/workflows/publish-container-images.yml (hunk 1 above)
Comment (jswank) [reply .github/workflows/publish-container-images.yml]: A previous PR created ops-server: this is the replacement DB backup process.

File: ansible/docker-files/keys/staging-1nfinite.pem
Comment (jswank) [reply ansible/docker-files/keys/staging-1nfinite.pem:1]: I'll clean the PR up a bit: there is no difference in the staging & prod PEM key, but I'd accidentally modified it in this branch.

File: ansible/group_vars/prod/secrets
Comment (jswank) [reply ansible/group_vars/prod/secrets:1]: Same as above: I need to clean up the PR a bit to remove this bit as well.

**Pull request closed without merging**